script.js

```diff
@@ -0,0 +1,9 @@
+function openNav() {
+    document.getElementById(
+      "myNav").style.width = "100%";
+}
+
+function closeNav() {
```

Comment:
- [ ] I would recommend you use arrow function here as it was introduced ES6 and it is the latest. It words different and better than the function key word.
- [ ] The close button, when clicked, it happens to navigate to the portfolio aspect of the page which means the close is inside the portfolio, kindly separate it so that the clash in functionality would be separated.

Reply:
Everything is fixed :+1: